packages/tools/webpack-fluid-loader/src/routes.ts

```diff
@@ -323,7 +323,7 @@ const fluid = (req: express.Request, res: express.Response, baseDir: string, opt
     <div id="content" style="min-height: 100%;">
     </div>
 
-    <script src="/fluid-loader.bundle.js"></script>
+    <script src="/code/fluid-loader.bundle.js"></script>
```

Comment:
>code [](http://example.com/codeflow?start=18&length=4)

nit: same here - Is this something that is broken ?

Reply:
https://github.com/microsoft/FluidFramework/pull/13519#discussion_r1068937131